terminate called after throwing an instance of 'SST::Core::Serialization::pvt::ser_buffer_overrun'
  what():  std::exception
[c28a-s2:26729] *** Process received signal ***
[c28a-s2:26729] Signal: Aborted (6)
[c28a-s2:26729] Signal code:  (-6)
[c28a-s2:26729] [ 0] /lib64/libpthread.so.0[0x3888c0f7e0]
[c28a-s2:26729] [ 1] /lib64/libc.so.6(gsignal+0x35)[0x38888325e5]
[c28a-s2:26729] [ 2] /lib64/libc.so.6(abort+0x175)[0x3888833dc5]
[c28a-s2:26729] [ 3] /apps/compilers/gcc/5.2.0/lib64/libstdc++.so.6(_ZN9__gnu_cxx27__verbose_terminate_handlerEv+0x15d)[0x2ab9e99f5b6d]
[c28a-s2:26729] [ 4] /apps/compilers/gcc/5.2.0/lib64/libstdc++.so.6(+0x8cbb6)[0x2ab9e99f3bb6]
[c28a-s2:26729] [ 5] /apps/compilers/gcc/5.2.0/lib64/libstdc++.so.6(+0x8cc01)[0x2ab9e99f3c01]
[c28a-s2:26729] [ 6] /apps/compilers/gcc/5.2.0/lib64/libstdc++.so.6(+0x8ce18)[0x2ab9e99f3e18]
[c28a-s2:26729] [ 7] sst[0x453f64]
[c28a-s2:26729] [ 8] sst[0x4cb63c]
[c28a-s2:26729] [ 9] sst(_ZN3SST15ConfigComponent15serialize_orderERNS_4Core13Serialization10serializerE+0xd25)[0x45e7a5]
[c28a-s2:26729] [10] sst(_ZN3SST11ConfigGraph15serialize_orderERNS_4Core13Serialization10serializerE+0x586)[0x461c76]
[c28a-s2:26729] [11] sst(_ZN3SST5Comms9serializeINS_11ConfigGraphEEESt6vectorIcSaIcEERT_+0xd1)[0x45ef21]
[c28a-s2:26729] [12] sst(_ZN3SST5Comms4sendINS_11ConfigGraphEEEviiRT_+0x1a)[0x45ef9a]
[c28a-s2:26729] [13] sst(main+0x86c)[0x4635fc]
[c28a-s2:26729] [14] /lib64/libc.so.6(__libc_start_main+0xfd)[0x388881ed1d]
[c28a-s2:26729] [15] sst[0x454239]
[c28a-s2:26729] *** End of error message ***
